File: exercicios/Python/1962/1962.py
n = int(input())

for i in range(n):
    t = int(input())

    if t > 2015:
        anos = (t - 2015) + 1
        print('{} A.C.'.format(anos))
    elif t < 2015:
        anos = (2015 - t)
        print('{} D.C.'.format(anos))
    
    elif t == 2015:
        print('1 A.C.')

File: exercicios/JavaScript/1962/1962.js
var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n');

var x = lines.shift()

for(let i=0; i<x; i++){
    var anos =0;
    var v = parseInt(lines.shift())

    if(v > 2015){
        anos = (v-2015)+1
        console.log(anos + ' A.C')
    }else if(v < 2015){
        anos = (2015 - v)
        console.log(anos + ' D.C')
    }
    else if(v == 2015){
        console.log('1 A.C')
    }
}